playbooks/templates/standalone_parameters.yaml.j2

```diff
@@ -59,7 +59,10 @@ parameter_defaults:
 {% if sriov_interface is defined %}
   NeutronPhysicalDevMappings: "hostonly:{{ sriov_interface }}"
   NovaPCIPassthrough: {{sriov_nova_pci_passthrough | mandatory | to_json }}
+{% elif pci_passthrough_devices is defined %}
```

Comment:
Why don't we re-use sriov_interface? What is configured with `sriov_interface`, that we don't want with `pci_passthrough_devices` ?

Reply:
we clarified it, and this is good to me.